src/command/subgraph/publish.rs

```diff
@@ -32,8 +36,12 @@ pub struct Publish {
     /// (often a deployed subgraph). May be left empty ("") or a placeholder url
     /// if not running a gateway or router in managed federation mode
     #[arg(long)]
-    #[serde(skip_serializing)]
     routing_url: Option<String>,
+    /// Url of a running subgraph that a supergraph can route operations to
+    /// (often a deployed subgraph). May be left empty ("") or a placeholder url
+    /// if not running a gateway or router in managed federation mode
```

Comment:
this should be updated, maybe something like this
```suggestion
    /// Disables the warning printed when publishing an invalid routing URL
```

Reply:
Oops didn't see your suggestion, but went with something a bit more verbose so I'll keep it.